grid/payments/delete-plan/profile.supr

```diff
@@ -0,0 +1,36 @@
+name = "payments/delete-plan"
+version = "1.0.0"
+
+"""
+Delete plan  usecase
+"""
+
+usecase DeletePlan unsafe {
+  input {
+    """
+    ID
+    Unique identifier of the plan to delete.
+    """
+    id! string!
```

Comment:
question: I think in some other profiles (subscription?) `planId` is used; I understand it is because it's a "foreign key" in those cases. Still, would you consider using `planId` as more explicit field name?

Reply:
Consistency above all, so I think `planId` or `productId` should be used everywhere. Same goes for primary ID in results.